resources/views/status.blade.php

```diff
@@ -42,6 +42,29 @@
                         {{$status->trainCheckin->HafasTrip->last_refreshed->diffForHumans()}}
                     </small>
                 @endif
+
+                @foreach($status->tags as $tag)
+                    @can('view', $tag)
+                        <span class="badge bg-trwl">
+                            @if($tag->key === 'trwl:seat')
```

Comment:
Maybe add an Enum with `\App\Interfaces\IconEnumInterface` to keep all icons at one central point and to reduce clutter?